### PR TITLE
Enable args in with_launch_file

### LINKED ISSUE #157: Add arg support for @with_launch_file
Launch files can take optional args:

```xml
<arg name="gzclient" default="false"/>

<group if="$(arg gzclient)">
    <node pkg="gazebo_ros" name="gazebo_gui" type="gzclient"/>
</group>
```

These can be set via launch files that include other launch files:
```xml
<include file="$(find buzzmobile)/launch/simulation.launch">
    <arg name="gzclient" value="true" />
</include>
```

And via command line:
```bash
$ roslaunch simulation.launch gzclient:=true
```

We'd like a way to set these in the test framework too:
```python
@with_launch_file('buzzmobile', 'simulation.launch')
# should also allow:
@with_launch_file('buzzmobile', 'simulation.launch', gzclient=True, world='something.world')
```

### PR DESCRIPTION
**Pull request by irapha**
Closes #157

### CODE REVIEW

File: buzzmobile/tests/simulation/test_gazebo_connection.py [hunk 1]
```diff
@@ -0,0 +1,17 @@
+import unittest
+import time
```
Comment (joshuamorton) [inline buzzmobile/tests/simulation/test_gazebo_connection.py]: remove unnecessary imports pls.

**Review by joshuamorton — CHANGES_REQUESTED** [review]

File: buzzmobile/tests/simulation/test_gazebo_connection.py (hunk 1 above)
Comment (irapha) [reply buzzmobile/tests/simulation/test_gazebo_connection.py]: why didn't style check catch this?

**Comment by joshuamorton** [comment]
your guess is as good as mine.

**Review by joshuamorton — APPROVED** [review]

**Comment by joshuamorton** [comment]
Do we want this in this file, or in a subfolder?

**Comment by irapha** [comment]
In this folder. Until we have a substantial amount of simulation tests, I don't foresee a good way to separate them out in subdirectories

**Pull request merged**